src/generator/CodeGenerator.php

```diff
@@ -26,20 +26,26 @@ public function __construct($config = null) {
 			$this->config = $config;
 		} else {
 			$this->config = new CodeGeneratorConfig();
-		} 
+		}
 
 		$this->init();
 	}
-	
+
 	protected function init() {
 		if ($this->config->getGenerateEmptyDocblock()) {
```

Comment:
The visitor strategy is still from the old package. Yet i'm fine with it. Nevertheless you are right, this would change this strategy. Continuing with multiple different visitors will end up in chaos. I'm happy to reduce it to one (as the visitor pattern is pretty good here). Passing the config to one visitor is fine.